components/event-processor/org.wso2.carbon.event.processor.core/src/main/java/org/wso2/carbon/event/processor/core/internal/storm/util/StormQueryPlanBuilder.java

```diff
@@ -210,6 +210,12 @@ private static Element constructPublisherElement(Document document,  String quer
      */
     private static void setAttributes(Element processor, String name, ParallelismInfoHolder holder) throws
             StormQueryConstructionException {
+
+        if (name.matches(".*\\s+.*")){
```

Comment:
Shall we enforce this at execution plan level for all query names and not specific to storm ? So it will be consistent.